### PR TITLE
Added code linter 

### PR DESCRIPTION
**Pull request by neelshah2409**
Added the linter.yml in workflows and updated the composer.

### CODE REVIEW

**Comment by neelshah2409** [comment]
@Meldiron please review it and the data.php thing once again prompted automatically so please see it .

**Comment by Meldiron** [comment]
@neelshah2409 I see, looks like automation is doing these changes. Let's ignore those.

File: composer.json [hunk 1]
```diff
@@ -18,7 +22,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.3",
-        "vimeo/psalm": "4.0.1"
+        "vimeo/psalm": "4.0.1",
+        "phpstan/phpstan": "1.9.x-dev"
```
Comment (Meldiron) [inline composer.json:26]: Wrong dependency. Linter uses `pint`, not `phpstan`. Let's switch that.

**Review by Meldiron — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (neelshah2409) [reply composer.json]: @Meldiron changes are done please review it .

**Review by Meldiron — APPROVED** [review]
Looks great 🎉

**Comment by christyjacob4** [comment]
@neelshah2409 please resolve the merge conflicts

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**